### ISSUE TITLE
`vcgGetEdge` and `vcgBorder` outputs are invisible

### ISSUE THREAD

**Issue by stla** [zarquon42b/Rvcg#39]
Hello,

The two functions mentionned in the title invisibly return their output. Is there a reason for that? I find this is a bit annoying.

**Comment by zarquon42b** [comment]
Hi, this is because the return large vectors/matrices and if you accidentally call the function without assigning the output you get printed a couple of 100.000 items. Why would you directly want to print that in your console? What usecase are you envisioning?
If you want to do this intentionally, you can still do:

```
edges <- vcgGetEdge(x)
edges
```

**Comment by zarquon42b** [comment]
Yes, but it will be too much to print and accidentally calling it will result in very long output.
